examples/05_deduplication.py

```diff
@@ -1,66 +1,67 @@
 """
-Deduplicating misspelled categories with deduplicate
-====================================================
+.. _examples_deduplication:
 
-Real world datasets often come with slight misspellings in the category
-names, for instance if the category is manually input. Such misspellings
-break many data-analyses steps that require exact matching, such as a
-'GROUP BY'.
+===================================
+Deduplicating misspelled categories
+===================================
 
-Merging the multiple variants of the same category or entity is known as
-*deduplication*. It is performed by the |dd| function.
+Real world datasets often come with misspellings in the category
+names, for instance if the category is manually inputted.
+Such misspelling break data analysis steps that require
+exact matching, such as a 'GROUP BY' operations.
 
-Deduplication relies on *unsupervised learning*, to find structure in
-data without providing explicit labels/categories of the data a-priori.
-Specifically clustering of the distance between strings can be used to
+Merging multiple variants of the same category is known as
+*deduplication*. It is implemented in skrub with the |deduplicate| function.
```

Comment:
The expansion of "|deduplicate|" is not happening right:
![image](https://github.com/skrub-data/skrub/assets/208217/4f615c3b-018c-4481-9735-b99756dcff31)